smartsim/_core/launcher/dragon/dragonConnector.py

```diff
@@ -245,9 +245,11 @@ def load_persisted_env(self) -> t.Dict[str, str]:
 
         with open(config.dragon_dotenv, encoding="utf-8") as dot_env:
             for kvp in dot_env.readlines():
-                split = kvp.strip().split("=", maxsplit=1)
-                key, value = split[0], split[-1]
-                self._env_vars[key] = value
+                # skip any commented lines
+                if not kvp.startswith("#"):
+                    split = kvp.strip().split("=", maxsplit=1)
```

Comment:
"=" will always exist? Do you need to raise if DNE?

Reply:
it will always exist. i create the file...